prereise/gather/flexibilitydata/doe/bus_data.py

```diff
@@ -0,0 +1,272 @@
+import errno
+import os
+import pickle as pkl
+import time
+from math import acos, cos, degrees, radians, sin
+
+import geopy
+import numpy as np
+import pandas as pd
+import requests
+import scipy.io as spio
```

Comment:
It looks like the `scipy.io` package can be removed now

Reply:
It appears that addressing this comment and my third comment would help, in part, to address the failing Lint test. If you look at the details of the failed test, you can see the other unused variables/packages that are causing the test to fail.